src/Table/Synapse/Assert.php

```diff
@@ -25,7 +25,7 @@ public static function assertTableIndex(string $indexName): void
     public static function assertValidClusteredIndex(string $indexName, array $indexedColumnsNames): void
     {
         if ($indexName === TableIndexDefinition::TABLE_INDEX_TYPE_CLUSTERED_INDEX
-            && count($indexedColumnsNames) !== 1
+            && count($indexedColumnsNames) === 0
         ) {
             throw new LogicException('CLUSTERED table index must have one key specified.');
```

Comment:
```suggestion
            throw new LogicException('CLUSTERED table index must have at least one key column specified.');
```

Vlastně ještě tohle...

Reply:
upraveno